nginx/default

```diff
@@ -6,7 +6,35 @@ server {
     #server_name euphoria-recruiting.club www.euphoria-recruiting.club;
     server_name localhost;
     location /api/ {
-        # insert Apollo IP address and server port number below
+        if ($request_method = 'OPTIONS') {
+           add_header 'Access-Control-Allow-Origin' '*';
+           add_header 'Access-Control-Allow-Methods' 'GET, POST, OPTIONS';
```

Comment:
you should include OPTIONS and DELETE here i think

Reply:
Do you mean 

```suggestion
           add_header 'Access-Control-Allow-Methods' 'GET, POST, OPTIONS, PUT, DELETE';
```

?

That didn't work.